src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -211,6 +213,7 @@ private String getInstructorQuestionResultsStatisticsHtml(
         df.setMinimumFractionDigits(0);
         df.setMaximumFractionDigits(5);
         df.setRoundingMode(RoundingMode.DOWN);
```

Comment:
Looks like `StringHelper.toDecimalFormatString` is not flexible enough. Let's create a `TeammatesDecimalFormat` for that?